open_academy/models/open_academy_course.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models, fields
+
+
+class Course(models.Model):
+    _name = "openacademy.course"
+    _description = "Course"
+
+    title = fields.Char(string="Course", required=True)
+    description = fields.Text()
+
+    responsible_user = fields.Many2one("res.users", ondelete="set null")
```

Comment:
In anothe PR please change this field. Follow the naming guidelines for Many2One fields

```suggestion
    responsible_user_id = fields.Many2one("res.users", ondelete="set null")
```